server/src/main/kotlin/models/Payment.kt

```diff
@@ -7,4 +7,12 @@ val paymentStorage = mutableListOf<Payment>()
 
```

Comment:
@jw122 one nit, should we move `/models` and `routes` into `com/redenvelope/flagship/`? It seems a bit unusual that they're not under the company domain